docs/cloud/index.md

```diff
@@ -25,15 +25,33 @@ You can [join the waitlist](https://pages.temporal.io/cloud-early-access).
 
 :::
 
+## How to
+
+- [Get started with Temporal Cloud](/cloud/how-to-get-started-with-temporal-cloud)
 - [Manage certificates in Temporal Cloud](/cloud/how-to-manage-certificates-in-temporal-cloud)
-- [Temporal Cloud release notes](/cloud/release-notes)
-- [Upgrade policy](/cloud/upgrade-policy-for-temporal-cloud)
+- [Manage certificate filters in Temporal Cloud](/cloud/how-to-manage-certificate-filters-in-temporal-cloud)
+- [Create a Namespace in Temporal Cloud](/cloud/how-to-create-a-namespace-in-temporal-cloud)
+- [Manage Namespaces in Temporal Cloud](/cloud/how-to-manage-namespaces-in-temporal-cloud)
 
 ## Tools
 
-- [What is tcld?](/cloud/tcld)
-- [How to install tcld](/cloud/tcld/how-to-install-tcld)
-- [tcld commands](/cloud/tcld/#tcld-commands)
+- [Temporal Cloud CLI (tcld)](/cloud/tcld)
+- [Temporal Web UI](/web-ui)
+
+## Information
+
+- [Account-level Roles](/cloud/what-are-the-account-level-roles-for-users-in-temporal-cloud)
+- [Namespace-level permissions](/cloud/what-are-the-namespace-level-permissions-for-users-in-temporal-cloud)
+- [Temporal Cloud release notes](/cloud/release-notes)
+
+## Upgrade policy
+
+All customers will automatically be upgraded to the latest minor version.
```

Comment:
I don't think this Upgrade Policy applies to the Cloud at all as the version we are running is managed by us and completely transparent to the end-user. I would just remove this.

Reply:
@rylandg, what's our current guidance on upgrade policy for Temporal Cloud?

Reply:
@mastermanu how would you specify SDK release updates as they relate to the upgrading we are doing?
While its true we havent changed our underlying api with sdk versions, we frequently questions about how our own upgrading impacts their SDK versioning.

Reply:
I think we can and should mention SDK compatibility with the Cloud. But this specific section doesn't appear to mention anything related to SDKs and seems to have been meant more for the server, which doesn't apply here

Reply:
So my understanding is that we are continuously upgrading Temporal Cloud, so this is an important statement. Cloud = OSS Server upgrades, so we should probably link out to a page that describes the Server / SDK compatibility caveats.